network/src/commonMain/kotlin/com/wire/kalium/network/api/user/client/ClientResponse.kt

```diff
@@ -38,6 +38,6 @@ data class ClientsOfUsersResponse(
 @Serializable
 data class SimpleClientResponse(
     @SerialName("id") val id: String,
-    @SerialName("class") val deviceClass: String
+    @SerialName("class") val deviceClass: String? = "unknown"
```

Comment:
1. There is already an enum `DeviceTypeDTO` for client class check `ClientResponse` 
2. do we need to default it to unknown, or we can leave it as null since I assume it's not used?

Reply:
> 1. There is already an enum DeviceTypeDTO for client class check ClientResponse

Oooh. Nice. Replacing with `DeviceTypeDTO`.


> 2. do we need to default it to unknown, or we can leave it as null since I assume it's not used?


I think defaulting to `Unknown` makes the fact that we tried, but the backend didn't answer us (or answered with something we don't understand) a bit more clear.

If the backend returns `spaceship`, it would be a bit weird to store it as `null`, also throwing an exception for something so little could be over-reacting.

Does it make sense?